deploy/ansible/roles/iroha-docker/templates/docker-compose.yml.j2

```diff
@@ -27,9 +27,10 @@ services:
     networks:
       - {{ iroha_network_name }}
       - iroha-db-net
+    restart: always
```

Comment:
add  `restart: always`  for postgres too.

also will be nice to have `mem_limit` as ansible variable. 
Usually for iroha we set:
```
mem_limit: 1024m
```

Reply:
> add restart: always for postgres too.

done
> also will be nice to have mem_limit as ansible variable.

Memory leaks are now fixed. Iroha memory consumption is now reduced greatly (like an orders of magnitude less). For internal projects, Soramitsu can use such a limit. Others should be able to tailor memory limit themselves.